config/samples/mongodb.com_v1_mongodbcommunity_cr_podantiaffinity.yaml

```diff
@@ -0,0 +1,58 @@
+---
+apiVersion: mongodbcommunity.mongodb.com/v1
+kind: MongoDBCommunity
+metadata:
+  name: example-mongodb
+spec:
+  members: 3
+  type: ReplicaSet
+  version: "4.2.6"
```

Comment:
[nit] can we put a newer version here? I think (outside the scope of this PR) we should periodically update this samples

Reply:
For the sake of consistency I suggest we do this in a follow up PR for all samples :)

Reply:
makes sense!